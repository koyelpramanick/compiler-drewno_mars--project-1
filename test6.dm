"backslash followed by space: \ is not allowed"
"\q"
"\\""
"unterminated
"test"
"\q \end"
"\n \q"
"testing // inside string literal"
"blah blah \" blah"
"'quote' in a string"
"@@@$$#$$##@@!!@#$$%%^&*()))(**&&^^%$$##@"
"unterminated string literal with \bad escape sequence
"bad escaped character: \a AND not terminated