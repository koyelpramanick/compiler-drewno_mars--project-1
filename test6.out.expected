STRINGLITERAL:"\\" [3,1]
STRINGLITERAL:"test" [5,1]
STRINGLITERAL:"testing // inside string literal" [8,1]
STRINGLITERAL:"blah blah \" blah" [9,1]
STRINGLITERAL:"'quote' in a string" [10,1]
STRINGLITERAL:"@@@$$#$$##@@!!@#$$%%^&*()))(**&&^^%$$##@" [11,1]
EOF [13,46]
